src/libraries/System.Data.OleDb/src/OleDbException.COMWrappers.cs

```diff
@@ -0,0 +1,60 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data.Common;
+using System.Diagnostics;
+using System.Globalization;
+using System.Runtime.Serialization;
+using System.Text;
+
+namespace System.Data.OleDb
+{
+    public sealed partial class OleDbException
+    {
+        internal static OleDbException CreateException(OleDbComWrappers.IErrorInfo errorInfo, OleDbHResult errorCode, Exception? inner)
+        {
+            OleDbErrorCollection errors = new OleDbErrorCollection(errorInfo);
+            string? message = null;
+            string? source = null;
+            OleDbHResult hr = 0;
+
+            if (null != errorInfo)
```

Comment:
Should we mark `OleDbComWrappers.IErrorInfo errorInfo` nullable?

Reply:
Actually not. This is uncovers unnecessary check here since `errorInfo` will be always not null. Should I remove check or mark as nullable?

Reply:
If this is confirmed to never be null removing the check is preferred.